### DISCUSSION TITLE
HOW TO DOWNLOAD IT

### DISCUSSION THREAD

**Discussion by Yan-Nunes** [apirrone/Memento#79] [Q&A]
Hey bro, congratulations on putting this idea into practice, could you make a video on YouTube showing how to download it?

**Comment by apirrone** [comment]
Hi @Yan-Nunes, thanks for your interest in Memento !

Currently, there is no "easy" way to download and install it, as this is still a prototype. I did not put the effort into packaging it as a simple executable.

However, it is not too hard to install assuming you are running linux. The "Installation" section of the readme is pretty self explanatory.
